mmv1/third_party/terraform/services/compute/resource_compute_subnetwork_test.go.tmpl

```diff
@@ -601,6 +632,22 @@ resource "google_compute_subnetwork" "network-with-private-google-access" {
 `, cnName, subnetworkName, cidrRange)
 }
 
+func testAccComputeSubnetwork_update4(cnName, subnetworkName string) string {
+	return fmt.Sprintf(`
+resource "google_compute_network" "custom-test" {
+  name                    = "%s"
+  auto_create_subnetworks = false
+}
+
+resource "google_compute_subnetwork" "network-with-migration-purpose" {
+  name          = "%s"
+  region        = "us-central1"
```

Comment:
This is missing here, causing the failure: Error: Error creating Subnetwork: googleapi: Error 400: Required field 'resource.ipCidrRange' not specified

```suggestion
  ip_cidr_range = "10.4.0.0/16"
  region        = "us-central1"
```